int_tests/sb_int_test.go

```diff
@@ -6,13 +6,16 @@ import (
 	"log"
 	"os"
 	"os/exec"
+	"strconv"
 	"testing"
 
 	yaml "gopkg.in/yaml.v2"
 
 	"github.com/stretchr/testify/assert"
 )
 
+var testRepo = "/tmp/strongbox-test." + strconv.Itoa(os.Getpid())
+
 func setUpCommand(dir, name string, arg ...string) {
```

Comment:
This is basically equivalent to `testCommand`, minus the returned output and with different failure modes. Why the two different functions? (although this is not part of this PR, I'm curious).

Reply:
No idea. Sounds like a good candidate for another refactor! :)

Reply:
This is already done, in the PR I raised yesterday and asked for feedback: https://github.com/uw-labs/strongbox/pull/14